pyproject.toml

```diff
@@ -116,9 +116,10 @@ files = [
     'scripts/**/*.py',
     # Incrementaly add files and paths to fix until the entire project is checked
 ]
-mypy_path = [
```

Comment:
what is the reason for this?

Reply:
The type sheet is not really working well with all python versions 3.8 - 3.10 (3.10 is not working properly).
Therefore I disabled typing for the pyodbc dependency.